tests/integration/relayer-tests/README.md

```diff
@@ -0,0 +1,51 @@
+# Relayer SDK Integration Tests
+
+This project contains tests for the OpenZeppelin Relayer SDK.
+
+## Setup
+
+To install dependencies:
+
+```bash
+bun install
+```
+
+## Running the Tests
+
+To run the tests:
+
+```bash
+RELAYER_ID={your id} API_KEY={your key} HOST_PORT={http://localhost:8080} bun test
+```
+
+```
+
+## Test Coverage
+As of now, we test the following methods:
+
+- `listRelayers`: shows all the relayers we have in the config. We should see our new chain here
+
+- `getRelayer`: shows a specific relayer’s info. We should see the info of our new chain.
+- `getRelayerBalance`: shows the native token balance of the relayer account. We should first fund it and check if we see the balance there.
+- `sendTransaction`: we send an empty “ping”-style EIP-1559 transaction that sends 0 ETH to the address 0xc834…db116, consuming only the intrinsic 21 000 gas for a plain transfer.
+- `getRelayerStatus`: shows status of relayer
+- `updateRelayer`: pause and resume relayer
+- `listTransactions`: shows the history of transactions and if they are pending, completed,
+
+Missing tests, we can add as needed:
+
+- `rpc`: NOT POSSIBLE for EVM as of now
```

Comment:
This is possible since https://github.com/OpenZeppelin/openzeppelin-relayer/pull/286 

Please include a simple RPC call for testing

Reply:
added in [91c7c38](https://github.com/librecapital/openzeppelin-relayer/pull/1/commits/91c7c38c2254cf8b98e7ab9c33f52f597b2a7c6b)